annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -542,14 +549,17 @@ protected void validateEdmRights(Body body)
      * black and white)
      *
      * @param webAnnotation
+     * @throws PropertyValidationException 
      */
-    private void validateTag(Annotation webAnnotation) throws ParamValidationException {
+    private void validateTag(Annotation webAnnotation) throws ParamValidationException, PropertyValidationException {
 	// webAnnotation.
 	Body body = webAnnotation.getBody();
 
 	// TODO: the body type shouldn't be null at this stage
 	if (body.getType() != null && body.getType().contains(WebAnnotationFields.SPECIFIC_RESOURCE)) {
 	    validateTagWithSpecificResource(body);
+	} else if (body.getType() != null && body.getType().contains(WebAnnotationFields.FULL_TEXT_RESOURCE)) {
```

Comment:
We don't have such a usecase, it must not be allowed to use fulltext resources when motivation is tagging. Please remove

Reply:
use case removed